webapp/store/views.py

```diff
@@ -183,10 +190,17 @@ def details_overview(entity_name):
     readme = md_parser(readme)
     readme = decrease_headers(readme)
 
+    topology = None
+    if package["type"] == "bundle":
+        topology = yaml.load(
+            package["default-release"]["revision"]["bundle-yaml"]
+        )
+
     return render_template(
         "details/overview.html",
         package=package,
         readme=readme,
+        topology=topology,
```

Comment:
If it's done in the `logic.py` then `package.store_front` could have the topology